lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+require 'order'
+class Menu
+
+  attr_reader :dishes, :order, :dishq
+
+  def initialize(dish_hash, dishes_quantity, order_class = Order)
+    @dishes = dish_hash
+    @order = order_class
+    @dishq = dishes_quantity
```

Comment:
Naming is confusing - I thought this was a typo when I saw it being used in the code!